src/chapters/sql-part2/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Library
+===============
+
+Word has gotten out about your amazing SQL skills!
+The head librarian at the local library asked if you could help with their database.
+
+Database Setup
+--------------
+
+Let's start by creating a new schema in MySQL Workbench called ``library``.
+Once you have accepted all defaults and hit "Apply", run the provided scripts to create tables and add data.
+By the end of your setup, you should have 6 tables:
+
+#. ``book``
+#. ``author``
+#. ``patron``
+#. ``reference_book``
+#. ``genre`` 
+#. ``loan``
+
+Loan Out a Book
+---------------
+
+A big function that you need to implement for the library is a query that updates the database when a book is loaned out.
+
+The query needs to perform the following functions:
```

Comment:
Since multiple tables need to be updated, will this function be a single query or a set of related queries?  Without knowing the columns in each table, I'm having trouble visualizing how to wrap this process into a single step.

Reply:
Either way, changed "query" to "script" so that people can accomplish the task however!